app/views/internal/articles/_individual_article.html.erb

```diff
@@ -134,8 +134,7 @@
               <input type="hidden" name="article_id" value="<%= article.id %>" />
               <p> Twitter MAIN</p>
               <textarea cols="37" rows="6" wrap="hard" name="tweet" maxlength="255"><%= article.title %>
-<% if !article.user.twitter_username.blank? %>&#013{ author: @<%= article.user.twitter_username %> } #DEVCommunity<% end %>
-              </textarea>
+<% if article.user.twitter_username %>&#013{ author: @<%= article.user.twitter_username %> } #DEVCommunity<% end %></textarea>
```

Comment:
I suggest using `twitter_username?` method to preserve the logic. `twitter_username` will be truthy in case it equals an empty string. Though we shouldn't have such values in the db I still suggest using the method with the question mark to be extra sure.

Reply:
@lightalloy Good caught! Fixed this.